packages/calcite-components/src/components/list/list.e2e.ts

```diff
@@ -1919,6 +1919,20 @@ describe("calcite-list", () => {
 
       const dMoveItems = await getMoveItems("d");
       expect(dMoveItems.length).toBe(0);
+
+      await page.evaluate(() => {
+        const firstLetters = document.getElementById("first-letters") as List["el"];
+        firstLetters.canPull = ({ dragEl }) => dragEl.id === "b";
+        firstLetters.canPut = ({ dragEl }) => dragEl.id === "c";
+        const secondLetters = document.getElementById("second-letters") as List["el"];
+        secondLetters.canPull = () => true;
+        secondLetters.canPut = () => false;
+      });
+      await page.waitForChanges();
+
+      expect((await getMoveItems("a")).length).toBe(0);
+
```

Comment:
Nitpick: assertions could be placed together since they are related and were updated by the same test action. Applies to `block-group` tests.